kp_pre_commit_hooks/gitops-values-validation.py

```diff
@@ -375,6 +391,31 @@ def validate_topic_name_compliance(self, value, schema):
         if match and match["serviceName"] != service_name:
             yield ValidationError(f"topicName '{value}' it not compliant, it should contain the service name '{service_name}'")
 
+        #maxLocalTopicBytesCompliance
+    def validate_max_local_topic_bytes_compliance(self, value, schema):
+
+        topics = value.get("managedResources", {}).get("mskTopics", {})
+        topic_env = value.get("env")
+        if not topics:
+            return
+        for topic_name, topic_config in topics.items():
+            topic_name = topic_config.get("topicName")
+            topic_max_local_bytes = topic_config.get("maxLocalTopicBytes")
+            print(f"topic_name: {topic_name}, topic_max_local_bytes: {topic_max_local_bytes}, env: {topic_env}")
+            if topic_name in WHITELISTED_TOPICS_FOR_MAX_LOCAL_TOPIC_BYTES:
+                whitelist_topic_config = WHITELISTED_TOPICS_FOR_MAX_LOCAL_TOPIC_BYTES[topic_name]
+                print(f"whitelist_topic_config: {whitelist_topic_config}")
+                if topic_env == whitelist_topic_config["env"]:
+                    if topic_max_local_bytes > whitelist_topic_config["max_limit"]:
+                        yield ValidationError(
+                            f"maxLocalTopicBytes '{topic_max_local_bytes}' for topic '{topic_name}' "
+                            f"exceeds the allowed maximum of {whitelist_topic_config['max_limit']} for environment '{topic_env}'"
+                        )
+                else:
+                    yield ValidationError(
+                        f"topic '{topic_name}' is not whitelisted for setting maxLocalTopicBytes in environment '{topic_env}'"
```

Comment:
We should always explain what to do in case of error, either directly or through a link to document that explain it if it's too long.
That's better for human but that will be good for AI too.